prereise/gather/flexibilitydata/test/test_geo_data.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+
+from geo.batch_process import *
+
+sys.path.append("..")
```

Comment:
I'm not sure why this is needed, but I bet there is a cleaner way to accomplish the same thing.

Reply:
Yeah I just found this line got moved by the formatter so the required file get added to path after the import attempt and caused an error. I am trying to find a workaround.

Reply:
What is the problem we're trying to solve with the `sys.path.append`?

Reply:
> What is the problem we're trying to solve with the `sys.path.append`?

Basically I am trying to import `batch_process.py` from `test_geo_data.py`, so I was trying to add the parent folder to sys.path so I can access the sub-folders.

flexibilitydata
|── geo
|   |── batch_process.py
|── test
|   |── test_geo_data.py

Reply:
https://github.com/Breakthrough-Energy/PreREISE/pull/282#discussion_r879941791